components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -9,6 +9,8 @@ import (
 const (
 	GCPPlanID         = "ca6e5357-707f-4565-bbbd-b3ab732597c6"
 	GCPPlanName       = "gcp"
+	GCPTrialPlanID    = "7d55d31d-35ae-4438-bf13-6ffdfa107d9f"
+	GCPTrialPlanName  = "gcp_trial"
```

Comment:
the name should be "trial"

Reply:
Wouldn't it be better to have the distinction between hyperscalers? I guess we might want to introduce a trial on different providers in the future.